src/command/client/import.rs

```diff
@@ -45,6 +45,11 @@ impl Cmd {
 
         match self {
             Self::Auto => {
+                if cfg!(windows) {
+                    println!("This feature does not work on windows.");
+                    return Ok(());
+                }
+
```

Comment:
In theory, we can compile away the auto arg completely on Windows. But tbh this is more self documenting so I like this better

Reply:
Another thing that we could be (probably not a good idea though) is to disable importing from shells that don't support windows like zsh. But it would just increase code complexity

Reply:
I'm happy with it being self-documenting, though could the message be a little clearer? Guide the user on what they _should_ do, not just tell them what they cannot

Ie "Auto import does not work on Windows, please specify your shell" or similar

Reply:
ok yea that would be better